popmon/pipeline/report_pipelines.py

```diff
@@ -181,9 +197,12 @@ def __init__(
         features=None,
         skip_empty_plots=True,
         last_n=0,
+        top_n=0,
```

Comment:
default is now set to 0, which is recognized as 20 later on. Might just as well put 20 here and omit the logic later